src/components/TextInput/index.tsx

```diff
@@ -244,8 +260,9 @@ const TextInput: VFC<TextInputType> = ({
           <BorderedWrapper
             borderedBackgroundColor={borderedBackgroundColor}
             borderedHeight={borderedHeight}
-            borderedColor={borderedColor}
+            borderedColor={focusBorderColor}
             borderedRadius={borderedRadius}
+            borderedWidth={borderedWidth}
```

Comment:
acho que nao precisamos do borderedWidth, o ideal é pegar o tamanho da borda pelo tema, e caso precise alterar que seja alterado no tema

Reply:
Não sei entendi, esse é o componente base, ele precisa dessa borda pra quando estiver no projeto do otmiza+ eu poder compor o componente, certo? Dai é esse momento que eu vou colocar os design tokens.

Reply:
@allefbuilders o fluid é utilizado seguindo o padrão de obter as cores, espaçamento e etc dos tokens..

Então em todo projeto que ele for instalado, ele vai pegar atraves do getTheme esses valores do Provider